### ISSUE TITLE
country not part of ad confirmation call - follow up to 8100

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#9823]
## Description 

Found while testing https://github.com/brave/brave-browser/issues/8100

Per above issue, `countryCode` should be present (if country is large enough) on release builds for ad confirmations.

## Steps to Reproduce

   1. Install 1.9.70 w/ flags(I used `--enable-logging=stderr --vmodule=brave_ads=3 --log-level=2`) and be running Charles
   2. Enable Rewards
   3. Trigger an ad

## Actual result:

build channel and platform are listed, but not countryCode:

```
[15741:775:0518/084541.203144:INFO:redeem_token.cc(108)] CreateConfirmation
[15741:775:0518/084541.203173:INFO:redeem_token.cc(110)] POST /v1/confirmation/{confirmation_id}/{credential}
[15741:775:0518/084541.203189:INFO:redeem_token.cc(113)] URL Request:
[15741:775:0518/084541.203208:INFO:redeem_token.cc(116)]   URL: https://ads-serve.brave.com/v1/confirmation/9330bfb8-0186-4884-864c-4aedf1cd4cab/eyJwYXlsb2FkIjoie1wiYmxpbmRlZFBheW1lbnRUb2tlblwiOlwiMXFBR1FWMXoyZSt2U2VRR0NjUTdOQk5WaTk3enJkRkhMc1loWHJzRXVudz1cIixcImJ1aWxkQ2hhbm5lbFwiOlwicmVsZWFzZVwiLFwiY3JlYXRpdmVJbnN0YW5jZUlkXCI6XCI4NGViMjc4NS05ZGViLTRhMmEtYTFjYS1mZjZjMjU3NzY4NWVcIixcInBheWxvYWRcIjp7fSxcInBsYXRmb3JtXCI6XCJtYWNvc1wiLFwidHlwZVwiOlwiZGlzbWlzc1wifSIsInNpZ25hdHVyZSI6IlRCN0FucEdyT09UYk5NcFBjUUdvblNWT01xRVZuWG1wTFBxRVZKUkg5R0V0TW5vUWx2MDJQNmtSQ2V4Z2R6elJacUZEcWY1cUR3dkJUMllZbFRUeHFRPT0iLCJ0IjoiWjkrU1ZvMTU2OWE2ekdrSU9JMUdPVXBDWFFmUWVCVTB5MDJEU29FRkJNRWx5Zmk5OTJXNnJmTmx2YkVucU16TEZPeUJBZWhmZlJoUkF2WVB5bUg5Y3c9PSJ9
[15741:775:0518/084541.203456:INFO:redeem_token.cc(128)]   Body: {"blindedPaymentToken":"1qAGQV1z2e+vSeQGCcQ7NBNVi97zrdFHLsYhXrsEunw=","buildChannel":"release","creativeInstanceId":"84eb2785-9deb-4a2a-a1ca-ff6c2577685e","payload":{},"platform":"macos","type":"dismiss"}
```

<img width="1083" alt="Screen Shot 2020-05-18 at 8 47 16 AM" src="https://user-images.githubusercontent.com/28145373/82228715-7c6bb900-98f7-11ea-807d-77454f272fc0.png">

## Expected result:
Since 1.9.70 is on Release channel and my region = US in my OS settings, I should have `countryCode` in the above, but it is not there.

## Reproduces how often: 

easily

## Brave version (brave://version info)

Brave | 1.9.70 Chromium: 81.0.4044.138 (Official Build) (64-bit)
-- | --
Revision | 8c6c7ba89cc9453625af54f11fd83179e23450fa-refs/branch-heads/4044@{#999}
OS | macOS Version 10.14.6 (Build 18G3020)

## Version/Channel Information:

- Can you reproduce this issue with the current release? 1.8.x n/a, but yes for 1.9.x release preview versions
- Can you reproduce this issue with the beta channel? not applicable - should only show for release builds
- Can you reproduce this issue with the dev channel? not applicable - should only show for release builds
- Can you reproduce this issue with the nightly channel? not applicable - should only show for release builds

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

cc @tmancey @jsecretan @rebron @brave/legacy_qa

**Comment by btlechowski** [comment]
Marked it as `QA/Blocked` till we get a release build.

**Comment by LaurenWags** [comment]
Removing `QA/Blocked` since https://github.com/brave/brave-browser/releases/tag/v1.11.94 is available

**Comment by LaurenWags** [comment]
Verification in progress (pending question)

```
Brave	1.11.94 Chromium: 84.0.4147.85 (Official Build) (64-bit)
Revision	e20c13e958edc3707db3d26d513780a7f6331a4b-refs/branch-heads/4147@{#837}
OS	macOS Version 10.14.6 (Build 18G3020)
```

* Verified STR from description
* Confirmed `buildChannel` and `platform` are still present.
* Spot checked countries to be included as per https://github.com/brave/brave-browser/issues/8100#issue-561129440. Confirmed for US (United States), NZ (New Zealand), and BE (Belgium) which are in the list from #8100.

<details>
<summary>Screenshots where countryCode is included</summary>

**countryCode = US (United States)**
<img width="521" alt="US - view" src="https://user-images.githubusercontent.com/28145373/87169122-8083d980-c29d-11ea-852e-5b6fab29d0ed.png">

**countryCode = NZ (New Zealand)**
<img width="491" alt="NZ - dismiss" src="https://user-images.githubusercontent.com/28145373/87169132-82e63380-c29d-11ea-801d-770e46580211.png">

**countryCode = BE (Belgium)**
<img width="537" alt="BE - click" src="https://user-images.githubusercontent.com/28145373/87169138-85488d80-c29d-11ea-975a-b656ba3d12bb.png">
</details>

* Confirmed a couple countries which are not included in the list in #8100 do not send country code. Ex: AW (Aruba), MC (Monaco) which are not in the list from #8100.

<details>
<summary>Screenshots where countryCode is not included</summary>

**countryCode = AW (Aruba)**
<img width="667" alt="Aruba - view" src="https://user-images.githubusercontent.com/28145373/87169155-8f6a8c00-c29d-11ea-9bf5-2686b0c97d8f.png">

**countryCode = MC (Monaco)**
<img width="665" alt="Monaco - click" src="https://user-images.githubusercontent.com/28145373/87169160-91344f80-c29d-11ea-8676-fcc30f3a2a28.png">
</details>

-----
